src/bundler.ts

```diff
@@ -409,7 +440,7 @@ class Bundler {
     });
 
     // We must clone the AST from the document, since we will be modifying it.
```

Comment:
Update comment or remove--?

Maybe: "We must parse document to a new AST since we will be modifying it."
Maybe also a note why cloneNode is insufficient to this task?